tests/dummy/app/components/smiley/component.js

```diff
@@ -0,0 +1,36 @@
+import Component from '@glimmer/component';
+import { tracked } from '@glimmer/tracking';
+
+export default class SmileyComponent extends Component {
+  @tracked x = this.args.x;
+  @tracked y = this.args.y;
+  @tracked radius = this.args.radius;
```

Comment:
These properties don't need to be tracked, as they are only assigned once and never change.
In the getters you can use `this.args.x` directly